tests/generator/README.md

```diff
@@ -0,0 +1,7 @@
+# Generate bundler p2p ssz object test cases
+
+## How to run generators
+
+```bash
+python main.py -o output_dir -c 10
```

Comment:
Please elaborate a bit, not only how to generate, but how to use, and preferably with code example if you can.